core/storage/statistics/gae_models.py

```diff
@@ -2066,8 +2066,11 @@ def get_all_models(
         else:
             return None
 
+    # Untyped decorator in cloud_transaction_services.py makes the function
+    # untyped. This ignore[misc] will be removed when the decorator has been
+    # type annotated.
     @classmethod
-    @transaction_services.run_in_transaction_wrapper
+    @transaction_services.run_in_transaction_wrapper # type: ignore[misc]
```

Comment:
Since `cloud_transaction_services.py` was not in mypy denylist, I removed unexplained usage of Any from it. Because of which this particular method became untyped in `cloud_transaction_services.py` and hence now requires a ignore here. Have added explanation above.